chaos-bugbounty-list.json

```diff
@@ -1752,6 +1752,26 @@
          "domains":[
             "pon-cat.com"
          ]
+      },
+      {
+         "name": "Playstation",
+         "url": "https://hackerone.com/playstation",
+         "bounty": true,
+         "domains":[
+            "playstation.net",
+            "sonyentertainmentnetwork.com",
+            "api.playstation.com",
+            "playstation.com"
+         ]
+      }
+      ,
+      {
+         "name": "EC-Council",
+         "url": "https://www.eccouncil.org/",
```

Comment:
Can you please update the URL with https://www.eccouncil.org/bug-bounty/

Reply:
done